ddtelemetry-ffi/src/worker_handle.rs

```diff
@@ -82,7 +88,39 @@ pub extern "C" fn ddog_handle_stop(handle: &TelemetryWorkerHandle) -> MaybeError
     MaybeError::None
 }
 
+#[allow(clippy::missing_safety_doc)]
+#[no_mangle]
+pub unsafe extern "C" fn ddog_handle_register_metric_context(
+    handle: &TelemetryWorkerHandle,
+    name: ffi::CharSlice,
+    metric_type: MetricType,
+    tags: ffi::Vec<Tag>,
+    common: bool,
```

Comment:
What is `common` ? what is it meaning ?
Could you add a docstring please ?

Reply:
Added 👍